src/main/resources/templates/workflow.html

```diff
@@ -160,7 +160,7 @@ <h2>Workflow: <span th:text="${workflow.label}">Workflow Name</span></h2>
 
         </div>
         <div class="col-lg-3 col-md-4 col-sm-6 hidden-xs">
-            <div class="graph-menu hidden-print" id="pull-right">
+            <div class="graph-menu hidden-print" id="graph-menu">
```

Comment:
See this file to confirm `graph-menu` is an ID, not a CSS class.

https://github.com/common-workflow-language/cwlviewer/blob/9c01b603c0e80f84af7c207ea9b7d59564260c97/src/main/resources/static/css/main-20180518.css#L168-L170